### DISCUSSION TITLE
পুরাতন নিয়মে স্বরবর্ণ লিখার সময় কীবোর্ড ফ্রীজ হয়ে যায়!

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#255] [General]
কোনও কোনও সফটওয়ারে, ব্রাউজারে অনেক সময় স্বরবর্ণ লিখতে গেলে ফ্রিজ হয়ে যায়। এটা ঘটে যখন আমি '্ + া' এভাবে লিখি তখন। মানে ফিক্সড লেআউটে ওল্ড স্টাইল টাইপিং মেথডে।
এটা র‍্যান্ডমলি হয়। নির্দিষ্ট সময় না। বেশিরভাগ ক্ষেত্রে ফেসবুকে, টুইটার ও মেট্রিক্স-এ (এলিমেন্ট ক্লায়ান্ট/ওয়েব এ্যাপ) হতো।
 বাগটা দেখা যায় যখন আমি '্+া' দিয়ে 'আ' লিখতে যাই (বিজয় বা ইউনিজয়ে), বা '্+'ি দিয়ে ই লিখতে গেলে। এরকম সব স্বরবর্ণে হয়। ততক্ষনাৎ কীবোর্ড ফ্রীজ হয়ে যায় (ইউআই ফ্রীজ না) । তারপর আবার অন্য জায়গায় ক্লিক করে এসকেপ চাপলে বা ibus এর লেঙ্গুয়েজ চ্যাঞ্জ করলে  কীবোর্ড কাজ করে।

Software version : 2.00 (Stable) and 3.00 (Development)
DE : KDE and Gnome
DM : Xorg and Wayland

**Comment by mominul** [comment]
আপনি কি `Kar Typing Order` ফিচারটিকে বোঝাচ্ছেন যেটি 3.0.0 (ডেভেলপমেন্ট) ভার্সনে যোগ করা হয়েছে?

**Reply by rashidi77** [reply]
না এটা স্ট্যাবল (২.০০) ভার্সনেও হতো। যখন আমি শব্দের শুরুতে ্+া=আ লিখতে যেতাম (উদাহরণস্বরূপ : আমার) তখন কীবোর্ড ফ্রীজ হয়ে যেতো। এটা হতো ফেসবুক আর এলিমেন্ট (মেট্রিক্স ক্লায়ান্ট)-এ বা সিমিলার ওয়েব এ্যাপ-গুলোতে। তবে আমি এখন ডেভেলপমেন্ট (৩.০০) ভার্সন ব্যবহার করছি। এটাতে এই সমস্যা প্রায় হয়না বললেই চলে। তাছাড়া ফেসবুকও ব্যবহার করছিনা, তাই ঠিক সমাধান হয়েছে কিনা জানা নেই। আমি ডিস্কাসনে এটা জানিয়ে রাখতে চেয়েছি। যেন বাগটা ধরতে পারলে সমাধান করা যায়।

**Reply by rashidi77** [reply]
It seems this bug still exists. This happens only when I turn 'Automatic Vowel Forming' off in fixed layouts. 

https://user-images.githubusercontent.com/93915281/144173651-2cc4844d-86ed-4161-bcd1-4a9d4b6fb292.mp4

I was trying to type ‘আমি’ but it was stuck on ‘আ’। I was typing ‘মি' but it was rejecting my inputs. Sometimes it replaces the ‘আ' with ‘মি'. Similar things happen with other vowels too.
